### PR TITLE
Release/2.3.0

### PR DESCRIPTION
**Pull request by sijeesh**

### CODE REVIEW

File: operators/hpe-csi-operator/deploy/operator.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ spec:
       containers:
         - name: hpe-csi-operator
           # Replace this with the built image name
-          image: quay.io/hpestorage/csi-driver-operator:v2.2.0
+          image: quay.io/hpestorage/csi-driver-operator:v2.3.0
```
Comment (datamattsson) [inline operators/hpe-csi-operator/deploy/operator.yaml:32]: I can't find this tag on Quay, is that intentional?

**Review by datamattsson — COMMENTED** [review]
e2e results on Ubuntu 22.04.1 with Kubernetes 1.26.1:

Nimble RWO:
```
Ran 91 of 7345 Specs in 4684.516 seconds
SUCCESS! -- 91 Passed | 0 Failed | 0 Pending | 7254 Skipped
PASS
```

Nimble RWX:
```
Ran 26 of 7347 Specs in 1306.368 seconds
SUCCESS! -- 26 Passed | 0 Failed | 0 Pending | 7321 Skipped
PASS
```

Primera RWO:
```
Ran 83 of 7345 Specs in 4186.586 seconds
SUCCESS! -- 83 Passed | 0 Failed | 0 Pending | 7262 Skipped
PASS
```

Primera RWX:
```
Ran 26 of 7347 Specs in 1359.039 seconds
SUCCESS! -- 26 Passed | 0 Failed | 0 Pending | 7321 Skipped
PASS
```

There's a problem with `PVCDataSource` on Primera that is unrelated to this release, hence disabled in the test spec and why you see less tests pass.

File: operators/hpe-csi-operator/deploy/operator.yaml (hunk 1 above)
Comment (sijeesh) [reply operators/hpe-csi-operator/deploy/operator.yaml:32]: Yes, wanted add all the changes required for the 2.3.0 releae in this PR. will be pushing the operator image once the QA appove the final build.

**Review by dileepds — APPROVED** [review]
LGTM

**Review by datamattsson — APPROVED** [review]
LGTM

**Pull request merged**